src/pykeen/models/base.py

```diff
@@ -1176,13 +1176,13 @@ def append_weight_regularizer(
         """
         # normalize input
         if isinstance(parameter, (str, nn.Parameter)):
-            parameter = list(parameter)
+            parameter = [parameter]
```

Comment:
@cthoyt will this also work for `Iterable[nn.Parameter]` like returned by `module.parameters()`?

Reply:
if the parameter is a string and you run list() on it, you get something you weren't expecting.

```python
>>> parameter = 'hello'
>>> list(parameter)
['h', 'e', 'l', 'l', 'o']
```

It won't even work if `isinstance(parameter, nn.Parameter)` becuase nn.Parameter isn't implementing the iterable interface (I think). If it's an iterable of parameters, then this code won't get run because it won't pass the isinstance check